### DISCUSSION TITLE
How to display trails on map around a national park

### DISCUSSION THREAD

**Discussion by AdminDevo** [rnmapbox/maps#1466] [Ideas]
I have only a latitude/longitude of a national park, i want to show also trails around the national park. but I dont have any lat/long of trail route. have anybody, an idea to display it on map. i want like attach a screenshots preview.
![Screenshot 2021-07-16 at 3 10 21 PM](https://user-images.githubusercontent.com/86459646/126027884-ae03a984-4403-487f-87ee-2fdc187d6d56.png)
![125927677-4e89fd75-ca02-42fc-8740-4c1c59685361](https://user-images.githubusercontent.com/86459646/126027903-0aa725fd-3e8c-49fa-b596-514431d7289f.png)

**Comment by ferdicus** [comment]
Without info of the trails, how are you supposed to display them?? 

You need to get the info of your route from somewhere and then render it over the map.

**Reply by AdminDevo** [reply]
thanks,
My question is, does the mapbox, provide routes of trails by any api based on national park lat/long.
(i have only park lat/long & wants to know routes of trails around each national park, any idea for getting this)
